common/src/main/java/com/microsoft/identity/common/internal/logging/IRequestContext.java

```diff
@@ -23,6 +23,6 @@
 package com.microsoft.identity.common.internal.logging;
 
 @Deprecated
-public interface IRequestContext extends com.microsoft.identity.common.logging.IRequestContext {
+public interface IRequestContext extends com.microsoft.identity.common4j.external.logging.IRequestContext {
```

Comment:
Not sure how i feel about the package name.  com.microsoft.identity.common.java instead?  Also is this the opportunity for the android library and the existing MSAL for Java library to start sharing code?

Reply:
1. That's the first name I picked 😆. Then I was suggested that common4j might be a better name (since we have msal4j). I don't have any strong preference over this.

2. I'd say that it's not our goal right now. We're not going to introduce any major changes in the code during the refactoring process. If MSAL4J wants to consume it in the future, then we can do it.

Reply:
Changed to com.microsoft.identity.common.java